README.md

```diff
@@ -37,7 +37,7 @@ exceptions. The most notable user visible differences, which might cause Idris
 + The `cong` function now takes its congruence explicitly as its first argument.
 + Minor differences in the meaning of export modifiers `private`, `export`,
   and `public export`, which now refer to visibility of names from other
-  *namespaces* rather than visibility from other *files*.
+  *namespaces* rather than the visibility from other *files*.
```

Comment:
Consider:
'name visibility' to clarify what visibility refers to, or
'visibility of names' to make consistent with the first part of the phrase.